test/e2e-encryption/encryption_test.go

```diff
@@ -1,7 +1,23 @@
 package e2e_encryption
 
-import "testing"
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	test "github.com/openshift/cluster-kube-apiserver-operator/test/library"
+)
 
 func TestEncryptionKeyEtcd(t *testing.T) {
-	t.Log("add tests here")
+	kubeConfig, err := test.NewClientConfigForTest()
+	require.NoError(t, err)
+	kubeClient, err := kubernetes.NewForConfig(kubeConfig)
+	require.NoError(t, err)
+	kv, done := test.NewEtcdKVMust(t, kubeClient)
+	defer done()
```

Comment:
it will panic when `done` is `nil`

Reply:
It will never be nil (this is a `Must` func that kills the test early)